src/renderer/widgets/CreateWallet/ui/MultisigWallet/components/WalletForm.tsx

```diff
@@ -80,7 +79,14 @@ export const WalletForm = ({ signatories, onContinue, isActive, isLoading, onGoB
     );
   });
 
-  const accountAlreadyExists = accounts.some((a) => a.accountId === multisigAccountId);
+  const accountAlreadyExists = wallets.some((wallet) => {
+    return (
+      !walletUtils.isWatchOnly(wallet) &&
```

Comment:
This approach seems quite overkill. Am I right that there is no easy way or util currently to go from an account to its wallet, beside filtering by `account.walletId`?

I would ideally check for an `account === multisigAccountId` and then only check whether this account is part of a wallet that is `watchOnly`. But I couldn't find an easy way to do this.